storybook/storyLoader.js

```diff
@@ -1,89 +1,92 @@
+
```

Comment:
some sort of prettier config gone haywire? I'm not sure what's correct, or how I might go about seeing/changing mine? I guess what I mean is, are these options automatically picked up per repo? I've sort of always been confused with prettier diffs that sometimes seem to go back and forth.

Reply:
Yeah, this is definitely ignoring the repo rules in the `package.json` - why, I'm unsure, the style here is the defaults without our config being respected. Perhaps you have an old version of prettier globally installed that takes precedence over the `node_modules` version?